pkg/injector/webhook.go

```diff
@@ -319,50 +322,46 @@ func patchAdmissionResponse(resp *v1beta1.AdmissionResponse, patchBytes []byte)
 	resp.PatchType = &pt
 }
 
-func patchMutatingWebhookConfiguration(cert certificate.Certificater, meshName, osmNamespace, webhookConfigName string, clientSet kubernetes.Interface) error {
-	if err := hookExists(clientSet, webhookConfigName); err != nil {
-		log.Error().Err(err).Msgf("Error getting MutatingWebhookConfiguration %s", webhookConfigName)
-	}
-	updatedWH := admissionv1beta1.MutatingWebhookConfiguration{
+// getPartialMutatingWebhookConfiguration returns only the portion of the MutatingWebhookConfiguration that needs to be updated.
+func getPartialMutatingWebhookConfiguration(cert certificate.Certificater, webhookName string) admissionv1beta1.MutatingWebhookConfiguration {
+	return admissionv1beta1.MutatingWebhookConfiguration{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: webhookConfigName,
+			Name: webhookName,
```

Comment:
Could you rename this to webhookConfigName? This was intentionally changed from webhookName previously to avoid confusing the mutatingwebhookconfiguration name from the webhook (L333) name.

Reply:
This change overall looks good, this comment is referring to a previous commit that renamed the webhook configuration name variable to `webhookConfigName`
```
    Resolve ambiguity between webhook config and webhook name (#1688)
    
    Currently the code uses the term 'webhook name' or the variable
    'webhookName' when in fact it is referring to the
    mutatingwebhookconfiguration name. This is confusing because
    the mutatingwebhookconfiguration has a webhook with a name, and
    it would be easy to confuse the two.
```